src/Compression/Utilities.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <city.h>
+
+#include <span>
+
+namespace DB
+{
+CityHash_v1_0_2::uint128 CalculateCityHash128InLittleEndian(std::span<char> buffer);
```

Comment:
Missing a comment.

The function name is incorrect.
It does not calculate something "in little-endian".

Poor separation of concepts in the code.
You want to write (serialize) the components of the hash in little-endian order. So, let's write the code in another way.

I can make it.

Reply:
I get your point around "in little-endian" being debatable, but it does indeed produce something in little-endian representation.

I also understand your concern that this has to do more with serialization than anything else and you don't wish to dilute other parts of the code with it.

Don't worry about it, I'll do it tomorrow.